src/strategies/sumami-holders/index.ts

```diff
@@ -0,0 +1,81 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'arugulo';
+export const version = '0.1.1';
+
+// Merged ABI for sUMAMI and Marinate contracts
+const abi = [
+  {
+    inputs: [{internalType: "address",name: "account",type: "address"}],
+    name: "balanceOf",
+    outputs: [{internalType: "uint256",name: "",type: "uint256"}],
+    stateMutability: "view",
+    type: "function"
+  },
+  {
+    inputs: [
+      {internalType: "address", name: "account", type: "address"},
+      {internalType: "uint32", name: "level", type: "uint32"}
+    ],
+    name: "stakedBalance",
+    outputs: [{internalType: "uint256",name: "",type: "uint256"}],
+    stateMutability: "view",
+    type: "function"
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  
+  const sUmamiBalances = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+
+  const marinateBalances = await Promise.all(
+    options.marinateLevels.map((level: number) =>
```

Comment:
We should have a limit of 4 marinateLevels here else some other space may pass in some big amount of requests into this
There is a limit of 5 calls
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy